providers/tests/apache/kylin/operators/test_kylin_cube.py

```diff
@@ -177,13 +177,15 @@ def test_render_template(self, session):
                 run_id="kylin_test",
                 logical_date=DEFAULT_DATE,
                 run_type=DagRunType.MANUAL,
+                state="running",
```

Comment:
Should we use `DagRunState.RUNNING` instead?

Reply:
Yeah, will update